README.md

```diff
@@ -40,6 +40,8 @@ cd uno
 eval "$(./bin/uno env)"             # Bash-specific command that sets up current shell
 ```
 
+Accumulo is now ready to use. Verify your installation by checking the status pages for the
```

Comment:
Could slightly shorten.  `Verify your installation by checking the [Accumulo Monitor](http://localhost:9995/) and [Hadoop NameNode](http://localhost:50070/) status pages.`  Not much of a shortening, just drops `for the`